backend/audit/templates/audit/remove-editor-access.html

```diff
@@ -0,0 +1,52 @@
+{% extends "base.html" %}
+{% load static %}
+{% block content %}
+    <div class="grid-container margin-top-6 margin-bottom-8">
+        <div class="grid-row">
+            <form class="tablet:grid-col-8 tablet:grid-offset-2"
+                  id="remove-editor-access"
+                  method="post">
+                {% csrf_token %}
+                <fieldset class="usa-fieldset">
+                        <h1 id="role-management">Remove Editor</h1>
+                        <p class="font-sans-md">
+                            {% if is_editor_removing_self %}
+                            You do not have permission to remove audit access for this editor:
+                            {% else %}
+                            Are you sure you want to remove audit access for this editor?
+                            {% endif %}
+                        </p>
+                        <p>
+                            <strong>Name: </strong>{{ name }}
+                        </p>
+                        <p>
+                            <strong>Email: </strong>{{ email }}
+                        </p>
+                        <input class="usa-input"
+                            id="editor_id"
+                            name="editor_id"
+                            type="hidden"
+                            value="{{ editor_id }}"
+                        />
+                        <span class="usa-error-message" id="email-error-message" role="alert">{{ errors.email }}</span>
+                </fieldset>
+                
+                {% if is_editor_removing_self %}
+                <p class="font-sans-md">
+                    In order to avoid an audit from become inaccesible to an organization, editors do not have permission to remove their own access.
```

Comment:
Simplify to "Editors do not have permission to remove their own access."

Reply:
Done!